pom.xml

```diff
@@ -154,10 +172,16 @@
         <artifactId>plexus-component-metadata</artifactId>
```

Comment:
Its deprecated plugin - do we still need this?

Reply:
Yes, so that I can disable the [plugin definition](https://github.com/codehaus-plexus/plexus-components/blob/master/pom.xml#L79-L89) in the parent pom.